README.md

```diff
@@ -12,6 +12,100 @@ Included:
 - Mythril
 
 
-To unbox:
+## To unbox:
 - Go to the desired location where you would normally run `truffle init`
-- Run `truffle unbox airswap/fluidity-truffle-box`
\ No newline at end of file
+- Run `truffle unbox airswap/fluidity-truffle-box`
+
+## Circle CI:
+A circle CI configuration file is provided already in the repository. This file is set up to perform:
+- `yarn install`
+- test migrations
+- run all tests and calculate their coverage
+- linting (this does not fix linting issues but reports them) 
+- clean up old branches in github
+
+## Migration Utils:
+This tool integrates with truffle migrations to enable a more comprehensive tracking of deployed contracts
+and their addresses. Utilising `migration_utils.js` enables multiple of the same contract to be deployed
```

Comment:
enables the same contract to be deployed multiple times with different settings

Reply:
Should we provide a 'how to use' the migration_utils.js?

Reply:
I think that sounds like a good idea Ethan. I'll finish the improvements deepa gave and then do you want to add that? I feel you understand it more than m

Reply:
yeah i'll add it.